docs/cookbook/rag.md

```diff
@@ -0,0 +1,7 @@
+# RAG (Retrieval-Augmented Generation)
+
+!!! note
+
+    Take a look at the [code](https://github.com/Mirascope/mirascope/blob/main/cookbook/rag_examples/local_embeddings.py) in our repo.
```

Comment:
same thing here, should we only link to local_embeddings.py or pinecone or the directory?